doc/Language/objects.pod6

```diff
@@ -447,10 +444,10 @@ After the C<BUILD> methods have been called, methods named C<TWEAK> are
 called, if they exist, again with all the named arguments that were passed
 to C<new>.
 
-Due to the nature of the default behavior of C<BUILDALL> and custom C<BUILD>
+Due to the default behavior of C<BUILDALL> and C<BUILD>
```

Comment:
How about:
The default behavior for C<BUILDALL> and C<BUILD> is to allow for named arguments in a C<new> method to correspond directly to the public attributes of any of the classes in a Method Resolution Order (MRO) or to any named parameter of any C<BUILD> submethod.  

from

Due to the default behavior of C<BUILDALL> and C<BUILD>
submethods, named arguments to the constructor C<new> derived from C<Mu> can
correspond directly to public attributes of any of the classes in the method
resolution order, or to any named parameter of any C<BUILD> submethod.